bintest/illegal_arguments_test.rb

```diff
@@ -6,17 +6,17 @@ def test_bad_command
     output, error, status = Open3.capture3(PATH, BIN, '-c="yabeda baba"',
                                            '-d=true', 'app')
     check_no_error(output, error, 'bad_command')
-    assert_true status.success?, 'Process did exit cleanly'
-    assert_include error, 'Process exited with status 127', 'return incorrect'
+    assert_false status.success?, 'Process did exit cleanly'
+    assert_include output, 'Process exited with status 127', 'return incorrect'
```

Comment:
Ist auch unter Windows gültig. Das problem ist, dass die meldung "Process exited with status 127" genau die Information ist, die ski von ssh.Run(cmd) als error erhält. 
(vollständiges objekt:
golang.org/x/crypto/ssh.Waitmsg: ssh.Waitmsg
lang: ""
msg: ""
signal: ""
status: 127)
jeglicher versuch die fehlermeldung weniger technisch zumachen wäre interpretation und damit in sich eine große fehlerquelle